public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" href="css/styles.css" type="text/css">
+
+<title>Fac Revs </title>
+</head>
+
+<body>
+<h1> Its working </h1>
```

Comment:
The server / handlers are working well. For the future though, it's more useful to do the skeleton of your site here instead. You can do this in another branch 💯